server/bin/unittests

```diff
@@ -257,6 +257,16 @@ function _verify_output {
     return ${res}
 }
 
+function _create_user {
+    name=${1}
+    # NOTE: we ignore failures here deliberately; it'll always fail in
+    # the 0[.x] tests because of intentionally corrupted configuration,
+    # and unintentional failure in other tests will result in visible
+    # symptoms later.
+    python3 ./lib/pbench/cli/server/user_create.py --username=${name} --email=${name}@example.com \
+        --first-name=${name^} --last-name=User --password=password >/dev/null 2>&1
```

Comment:
It's the pragmatic straightforward way to do this, though!

Yeah, it might be "fun" to make the `unittests` script do a local install of `pbench-server` under its `/var/tmp` path and use that to allow us to access the full click-ified commands; but all we need for the "traditional" commands is `PYTHONPATH` and `PATH`, which we have... so that's a substantial amount of new work to make this one bit look a bit cleaner. I decided that effort wasn't justifiable right now.

Reply:
> all we need for the "traditional" commands is `PYTHONPATH` and `PATH`

That and all the stuff in the `requirements.txt` and/or `setup.py`...which, by the way, is all that is needed to set up the Click commands.

(I think a `pip install .` will do it...I'm not sure what that "it" is -- I do it in a Python "virtual environment", but I assume that the unit tests are already doing something similar -- but, after issuing that command, then the Click tools are immediately accessible via my shell path.)